product_docs/docs/biganimal/release/administering_cluster/notifications.mdx

```diff
@@ -1,32 +1,44 @@
-| Organization | Personal access key is expiring                                                                  | Account owner                    | All                  |
-| Organization | Machine user access key is expiring                                                              | Organization owner               | All                  |
-| Project      | Upcoming maintenance upgrade on a cluster in 24 hours                                          | Project owner/editor             | All                  |
-| Project      | Successful maintenance upgrade on a cluster                                                      | Project owner/editor             | All                  |
-| Project      | Failed maintenance upgrade on a cluster                                                          | Project owner/editor             | All                  |
-| Project      | Paused cluster will automatically reactivate in 24 hours                                         | Project owner/editor             | All                  |
-| Project      | Paused cluster was automatically reactivated                                                     | Project owner/editor             | All                  |
-| Project      | You must set up the encryption key permission for your CMK-enabled cluster                       | Project owner/editor             | All                  |
-| Project      | Key error with CMK-enabled cluster                                                               | Project owner and project editor | All                  |
-| Project      | User is invited to a project (displays only to the Project owner)                                | Project owner                    | All                  |
-| Project      | New role is assigned to you                                                                      | Account owner                    | All                  |
-| Project      | Role is unassigned from you                                                                      | Account owner                    | All                  |
-| Project      | Failed connection to third-party monitoring integration (and future non-monitoring integrations) | Project owner/editor             | All                  |
+Different types of events are sent as notifications. Users with different roles can configure the preferences to receive these notifications in the in-app inbox, by email, or both. 
+The notifications are categorized into following preference sections:
+- Account
+- Organizations
+- Projects
+
+The notifications under **Account** preference section:
+
+- Project role unassigned from you
+- Personal access key is about to expire
+- New project role assigned to you
+
+The notifications under **Organizations** preference section:
+
+- Payment method added (specific to organizations that opted credit card payment option)
+- Machine user API key is expiring
```

Comment:
`Machine user API key is expiring` --> `Machine user access key is expiring` 

I think we never use term of `API key` since the Access Key feature is released.